src/rules/javascript/eval_string_arg.js

```diff
@@ -0,0 +1,17 @@
+export default function(context) {
+  return {
+    CallExpression: function(node) {
+      var acceptedArgs = ['FunctionExpression', 'ArrowFunctionExpression'];
+
+      if (node.callee.property.type === 'Identifier' &&
+          (node.callee.property.name === 'setTimeout' ||
+           node.callee.property.name === 'setInterval')) {
+
+        if (node.arguments.length > 0 &&
+            acceptedArgs.indexOf(node.arguments[0].type) === -1) {
+          return context.report({node: node, message: 'eval_string_arg'});
```

Comment:
This should import the message you're using and use the code prop in place of the `'eval_string_arg'`. We're going to add a test for this soon so all new js rules could start doing that now to save us needing to refactor this later.